code/modules/admin/admin.dm

```diff
@@ -1326,3 +1326,19 @@ proc/admin_notice(var/message, var/rights)
 	message_admins("[key_name(usr)] attempting to force mode latespawn.")
 	ticker.mode.next_spawn = 0
 	ticker.mode.try_latespawn()
+	
+/datum/admins/proc/set_tcrystals(mob/living/carbon/human/H as mob)
+	set category = "Debug"
+	set name = "Set Telecrystals"
+	set desc = "Allows admins to change telecrystals of a user."
+	
+	var/crystals
+	
+	if(R_ADMIN)

```

Comment:
I'd like to mention that Neerti caught it prior to server merge. I was also kind of out of it at the time.